config/logging.yaml

```diff
@@ -0,0 +1,83 @@
+# enable logging output inline in rails console
+:console_inline: false
+# colorize log including level token based on severity
+:colorize: false
+# adds an extra line after each log message with information from which file:line and method
+# was this message logged
+:log_trace: false
+#do not log certain paths (example  /notices/get_new)
+:ignored_paths: []
+# configuration of all loggers used in app
+:loggers:
+  # this logger is parent of all other loggers, by default this is the only one having appender
+  # which is used by all other loggers
+  :root:
+    # only messages with severity >= this level from all loggers will be logged
+    # valid values:  debug, info, warn, error, fatal, unknown
+    :level: warn
+    # file or syslog, for file variant you can specify other options below like filename,
+    # pattern or rolling options
+    :type: file
+    # 'daily', 'weekly', 'monthly' or number as seconds
+    :age: 'weekly'
+    # how many old log files to keep (156 = 52 * 3 = 3 years for weekly age)
+    :keep: 156
+    # logger line pattern, you can use following macros
+    #  [%c] name of the logger that generate the log event
+    #  [%d] datestamp
+    #  [%m] the user supplied log message
+    #  [%p] PID of the current process
+    #  [%r] the time in milliseconds since the program started
+    #  [%T] the name of the thread Thread.current[:name]
+    #  [%t] object_id of the thread
+    #  [%F] filename where the logging request was issued
+    #  [%L] line number where the logging request was issued
+    #  [%M] method name where the logging request was issued
+    #  [%X{string}] variable set using ::Logging.mdc['string'] =
+    :pattern: "%d [%c] [%.1l] %m\n"
+    # log filename for this environment, it will be placed into #{Rails.root}/log directory
+    # note when you log outside of standard rails path you won't see log output in WEBrick
+    # STDOUT (usually not wanted in development), to use Rails default path, don't set path
+    # path: /var/log/katello
+    :filename: "production.log"
+
+  # you can configure each logger individually, you can turn it off completely by using key
+  # 'enabled' or you can adjust it's logging 'level'. If you specify level higher than root
+  # logger has, messages below this level are ignored. If you specify lower level than root
+  # they are still ignored but you can add appender in Foreman::Logging just for this logger
+  # and see these logs outside of main log.
+  :app:
+    :enabled: true
+  :sql:
+    :enabled: true
+
+:production:
+  :loggers:
+    :sql:
+      :level: fatal
+
+:development:
+  :ignored_paths: ['/assets/']
```

Comment:
Looks like this was actually a custom solution for allowing ignoring any paths through middleware -- https://github.com/Katello/katello/blob/master/lib/katello/middleware/silenced_logger.rb . If support for this generic ignoring is something we want I can port this middleware over, otherwise I can remove this aspect and keep quiet_assets.